pandas/tests/scalar/test_timedelta.py

```diff
@@ -13,6 +16,135 @@
 
 
 class TestTimedeltaArithmetic(object):
```

Comment:
as I said in another PR, pls create a 

``pandas/tests/scalar/timedelta/test_arithmetic`` to add these 2

Reply:
let's do that *first* so that its then easier to see the changes in this.

Reply:
OK.  Will open a new PR that just _moves_ tests without making any actual changes.

Reply:
Edit: it will add tests that currently pass, will not _remove_ any existing tests or touch anything outside of the tests.